library/MiscUtils.cpp

```diff
@@ -168,32 +168,58 @@ std::string to_search_normalized(const std::string &str)
     return result;
 }
 
-bool word_wrap(std::vector<std::string> *out, const std::string &str, size_t line_length)
+
+bool word_wrap(std::vector<std::string> *out, const std::string &str,
+               size_t line_length, bool collapse_whitespace)
 {
-    out->clear();
-    std::istringstream input(str);
-    std::string out_line;
-    std::string word;
-    if (input >> word)
+    if (line_length == 0)
+        line_length = -1;
```

Comment:
I would suggest `SIZE_MAX` as a slightly more clear alternative to the underflow of -1 here

Reply:
Thanks, done.